src/Uno.UI/ts/Windows/Media/SpeechRecognizer.ts

```diff
@@ -0,0 +1,98 @@
+﻿interface Window {
+	SpeechRecognition: any;
+	webkitSpeechRecognition: any;
```

Comment:
If possible, you should try to use right types instead of `any`

Reply:
Unfortunately the type bindings for Speech recognition are not yet finalized, so I resorted to this for now, once the types are finalized, I will update accordingly